ios/22/_sentry_arm64.crash

```diff
@@ -0,0 +1,35 @@
+OS Version: iOS 10.2.1 (14D27)
+Report Version: 104
+
+Exception Type: EXC_BREAKPOINT
+Crashed Thread: 0
+
+Application Specific Information:
+Exception 6, Code 771328, Subcode 8
+
+Thread 0 name:
+Thread 0 Crashed:
+0   CrashLibiOS                     0x1000bc500         @objc CRLCrashSwift.crash() -> ()
+1   CrashLibiOS                     0x1000bc500         CRLCrashSwift.crash() -> () (CRLCrashSwift.swift:36)
```

Comment:
There can't be two frames with the same memory address and different results. This should be shown as one frame with the combined result. Otherwise this should be read by the user as frame 0 is called by frame 1. Which is impossible with the given data.